redis/Cargo.toml

```diff
@@ -60,6 +60,9 @@ backon = { version = "1.4.0", optional = true, default-features = false }
 # Only needed for the r2d2 feature
 r2d2 = { version = "0.8.10", optional = true }
 
+# Only needed for the bb8 feature
+bb8 = { version = "0.9.0", optional = true }
```

Comment:
please add this as an explicit feature, and add a rule in config/redis.toml that bb8 & r2d2 won't be built at the same time.

Reply:
> bb8 & r2d2 won't be built at the same time.

Hi, technically `bb8` be used alongside `r2d2` at the same time. Is there any concern about this?

Reply:
it's not a product consideration, just a heuristic to reduce test time. The `flag-frenzy` action checks an increasing number of combinations, and in order to reduce the time that step takes, I want to remove unlikely combinations from the checks. Our lint and test stages test with all feature enabled, so I'm doubtful that there's an interaction there that might be missed.